src/components/PlioQuestion.vue

```diff
@@ -95,6 +95,13 @@
           आगे बढ़ें
           </button>
         </div>
+        <progress-bar 
+          v-if="isProgressBarEnabled"
+          ref="progressBarRef"
+          :startPercent="progressBarInfo['completionPercent']"
+          :config="progressBarInfo['config']"
+        >
+        </progress-bar>
```

Comment:
Progress bar component.
- Will show only when it's enabled. (this `enabled` info comes from the `Player.vue` which fetches the plio config for this specific plio)
- `startPercent` -> this tells the progress bar to start at `x %` whenever it is rendered. 
- `config` -> this is the specific progress bar details that have been fetched by the backend

Reply:
1. This is specifically a question progress bar no? 
2. This config doesn't have default values in the default config?
3. Naming mismatch - `startPercent` vs `completionPercent`. Maybe rename both to `progressPercent`?